plugins/generic/announcementFeed/templates/block.tpl

```diff
@@ -14,17 +14,17 @@
 		<ul>
 			<li>
 				<a href="{url router=$smarty.const.ROUTE_PAGE page="gateway" op="plugin" path="AnnouncementFeedGatewayPlugin"|to_array:"atom"}">
-					<img src="{$baseUrl}/plugins/generic/announcementFeed/templates/images/atom10_logo.gif" alt="{translate key="plugins.generic.announcementfeed.atom.altText"}">
+					<img src="{$baseUrl}/plugins/generic/announcementFeed/templates/images/atom.svg" alt="{translate key="plugins.generic.announcementfeed.atom.altText"}">
```

Comment:
Rather than duplicating these images, can you refactor to use the images located in `pkp-lib`? The URL would be `{$baseurl}/lib/pkp/templates/images/atom.svg}`.

Same for the others below.